doc/user/content/installation/install-on-aws/_index.md

```diff
@@ -59,6 +59,10 @@ for details.
 If you do not have Helm 3.2.0+, install. For details, see the [Helm
 documentation](https://helm.sh/docs/intro/install/).
 
+### License key
+
+To run Materialize in Kubernetes, a license key is required. To request a license key for Materialize Community Edition, visit [https://materialize.com/get-started/].
+
```

Comment:
Could we add this blurb to a file in doc/user/shared-content/ ?

Then, all we need to do in the various files is 
`{{< include-md file="shared-content/filename.md" >}}`

Reply:
does this look right? i'm not sure how to build/preview the docs

Reply:
how does this look?

Reply:
Kind of like "If you have purchased an Enterprise License, your license key can be obtained blah..."

Reply:
i think that's still kind of up in the air - for now, i think we're just going to generate them internally and email them to users or something along those lines. i think we can update the docs later once we have a better idea of what the workflow will look like.